src/main/java/edu/carrollcc/cis232/Question2.java

```diff
@@ -1,10 +1,42 @@
 package edu.carrollcc.cis232;
 
+import java.util.StringTokenizer;
+
 public class Question2 {
 
 	public static void main(String[] args) {
-
+		String palindrome = "RaCecar";
+		
+		StringTokenizer tokens = new StringTokenizer(palindrome, " !@#$%^&*()_-+.,><?/'`~{}[]");
+		
+		while(tokens.hasMoreTokens()){
+			String token = tokens.nextToken();
+			System.out.printf(palindrome, token);
+			
+			if(isPalindrome(palindrome.toLowerCase()))
+	            System.out.println(palindrome + " is a palindrome");
+	        else
+	            System.out.println(palindrome + " is not a palindrome");
+		}
 	}
 	
 	//write static isPalindrome method here
+	public static boolean isPalindrome(String palindrome){
```

Comment:
Does not pass capitalization or punctuation tests (3 unit tests failed -9 technical)